src/pages/pages/login/index.js

```diff
@@ -232,9 +236,15 @@ const LoginPage = () => {
 
               // onClick={() => router.push('/')}
               onClick = {
-                (e) => {
+                async (e) => {
                   e.preventDefault(); 
-                  sendData(); 
+                  // sendData(); 
```

Comment:
I think here it would be greater if we use a state that contains the needed data (email, password) because before sending data we will need to do some verifications